File: SourceCode/C/day020.c
/* Author: Ahmed A. M. ALOMAR
Date: January 20 - 2020
Description:
This program is an example for the data type Conversions */


#include <stdio.h>

int main()
{
  int a = 2.4321;
  int x = 5;
  int y = 2;
  float z;

  // the (float) is the casting operator
  z = (float)x / y;
  printf("z = %f \n", z);

  // Converting a float into an integer
  printf("%d \n", a);

  return 0;
}

File: SourceCode/Python/day020.py
# ------------------------------------ #
# Author: Ahmed A. M. ALOMAR
# Date: April 30 - 2020
#
# Description:
# * Example of string methods
# ------------------------------------ #


# Some string
reminder = "To find fundamental limits on signal processing operations such as compressing data and on reliably storing and communicating data."


# .find()
print(reminder.find("reliably"))

# .count()
print(reminder.count('e'))

# .replace()
print(reminder.replace("data", "chiken"))

# .format()
letter = "{} is very {} and {}."
print(letter.format("Dinner", "delicious", "satisfying"))
print(letter.format("This rose", "colorful", "beautiful"))
